config/i18n.json

```diff
@@ -485,9 +485,8 @@
     "ThumbsUp": "Thumbs Up",
     "ThumbsDown": "Thumbs Down",
     "SetBonus": {
-      "2Piece": "2 Piece",
-      "4Piece": "4 Piece",
-      "NPiece": "{{count}} Piece"
+      "NPiece_one": "{{count}} Piece",
+      "NPiece_other": "{{count}} Pieces"
     }
```

Comment:
Worth noting that in-game they use singular for specifically these
<img width="1119" height="844" alt="image" src="https://github.com/user-attachments/assets/3b5a3d71-fa4e-4647-b63e-f1f7fb3dc3a6" />

Reply:
OK. I'm not comfortable assuming that's true for all languages though, so we'll need to still translate it as a plural.